### PR TITLE
update dedicated ROADMAP.md file

### PR DESCRIPTION
**Pull request by orville-wright**
Create dedicated ROADMAP.md file for completeness as its referenced in README.md but should be its own file.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,56 +1,109 @@
 # Roadmap
 
-This document provides information on OpenEBS development in current and upcoming releases. Community and contributor involvement is vital for successfully implementing all desired items for each release. We hope that the items listed below will inspire further engagement from the community to keep OpenEBS progressing and shipping exciting and valuable features.
+This Roadmap defines OpenEBS features and capabilities that are in current development and may be incldued in upcoming releases.<BR>
```
Comment (avishnu) [inline ROADMAP.md:3]: included*

File: ROADMAP.md [hunk 2]
```diff
@@ -1,56 +1,109 @@
...
-These are some of the backlogs that are prioritized and planned to be completed within the next major release (e.g. OpenEBS 4.0). While the following are planned items, higher priority is given to usability and stability issues reported by the community. The completion of these items also depends on the availability of contributors.
+## Active Master roadmap
+This table holds a list of the most active roadmap items. These are the features that are currently getting the most active focus and attention within the project.
 
-Note: OpenEBS follows a quarterly release cadence with a new minor release around the end of each quarter. For the most current plan and status check out the [release project trackers](https://github.com/orgs/openebs/projects?type=classic). This document is reviewed and updated by the maintainers after each major release.
-
-### Mayastor
-- Source repositories
+## Current
+These are backlogs items that are prioritized and planned to be completed within the next major release. While many are planned items, higher priority is given to usability, stability, resilence, Data integrity issues reported by the community.
```
Comment (avishnu) [inline ROADMAP.md:14]: backlog

File: ROADMAP.md [hunk 3]
```diff
@@ -1,56 +1,109 @@
...
+| 12   | Thin provisioning phase-2                     | Thin Provision awareness and integrations with DiskPool metrics, pre-emptive intelligence actions                   | Pri 3 /  Rel: (Q4 2024)    |
+| 13   | Native Object Store                           | An S3-compliant fast object store   integrated with SPDK LVstore/LVols Blobstor & HyperLocal-PV vols               | Pri 3 /  Rel: (Q4 2024)    |
+| 14   | Zoned-SSD support                             | Integrated Western Digital Team's Mayastor ZNS feature for very high performance vols                             | Pri 2.5 /  Rel: (Q2 2024)   |
+<BR>
+
+## Excluded from the roadmap
+The Roadmap is focused exclusively on the modern **Mayastor* Data-Engines in the **STANDARD** edition. 
+<BR>
+It does not define any net-new features or capabilities for any OpenEBS **LEGACY** projects or projects that are tagged & defined as DEPRERCATED or ARCHIVED. (Currently those projects are as follows):<BR>
+| ID  | Data-Eegines      | Embeded tech stack   | Status                           |
```
Comment (avishnu) [inline ROADMAP.md:54]: Engines*
Embedded*

File: ROADMAP.md [hunk 4]
```diff
@@ -1,56 +1,109 @@
...
+| 11   | E2E Storage UNMAP reclaim integration          | Support Discard: LINUX / UNMAP: SCSI / Deallocate: NVMe issued from filesystem down to SPDK Blobstor elements    | Pri 3 /  Rel: (Q4 2024)    |
+| 12   | Thin provisioning phase-2                     | Thin Provision awareness and integrations with DiskPool metrics, pre-emptive intelligence actions                   | Pri 3 /  Rel: (Q4 2024)    |
+| 13   | Native Object Store                           | An S3-compliant fast object store   integrated with SPDK LVstore/LVols Blobstor & HyperLocal-PV vols               | Pri 3 /  Rel: (Q4 2024)    |
+| 14   | Zoned-SSD support                             | Integrated Western Digital Team's Mayastor ZNS feature for very high performance vols                             | Pri 2.5 /  Rel: (Q2 2024)   |
+<BR>
+
+## Excluded from the roadmap
+The Roadmap is focused exclusively on the modern **Mayastor* Data-Engines in the **STANDARD** edition. 
+<BR>
+It does not define any net-new features or capabilities for any OpenEBS **LEGACY** projects or projects that are tagged & defined as DEPRERCATED or ARCHIVED. (Currently those projects are as follows):<BR>
```
Comment (avishnu) [inline ROADMAP.md:53]: DEPRECATED*

File: ROADMAP.md [hunk 5]
```diff
@@ -1,56 +1,109 @@
...
-- Backlogs (near-term)
-  - Support for hot upgrade, business continuity with no downtime.
-  - Support for volume groups, i.e. Mayastor replica placement should be topology aware for statefulsets zonal (or HA) distribution.
-  - Support for Mayastor volume resize.
-  - Support for volume snapshots.
 
-### Dynamic Local PVs
-- Source repositories
+### Dynamic Local PVs repos
+- (These repos are integating into Mayastor to be included as part of the Mayastor STANDARD edition - Q1 2024)
```
Comment (avishnu) [inline ROADMAP.md:70]: integrating

File: ROADMAP.md [hunk 6]
```diff
@@ -1,56 +1,109 @@
...
   - https://github.com/openebs/lvm-localpv
   - https://github.com/openebs/rawfile-localpv
   - https://github.com/openebs/device-localpv
   - https://github.com/openebs/node-disk-manager
+
 - Backlogs
-  - Shared VG for LVM Local PV. 
+  - Shared VG for LVM Local PV.
+  
+## Backlogs in-progress (under active development)
```
Comment (avishnu) [inline ROADMAP.md:81]: Old set, already completed.
Need to be updated with latest in-progress items

File: ROADMAP.md [hunk 7]
```diff
@@ -1,56 +1,109 @@
...
+  - Removal of iSCSI support from Mayastor, NVMe is the only supported protocol.
+  - On manual deletion of a Mayastor PV having Reclaim policy set to "Retain", the related Mayastor resources need to be cleaned up.
+  - High availability support for Mayastor nexus (target), ability to spin up on-demand replacement target for a Mayastor volume.
+  - Faster rebuild of volume replicas using log-based technique.
 
+## Backlogs (near-term)
```
Comment (avishnu) [inline ROADMAP.md:87]: Old set, already completed. Need to be updated with latest near-term items

File: ROADMAP.md [hunk 8]
```diff
@@ -1,56 +1,109 @@
...
+### Others
+  - https://github.com/openebs/charts
+  - https://github.com/openebs/openebsctl
+  - https://github.com/openebs/monitoring
+  - https://github.com/openebs/website
+  - https://github.com/openebs/m-exporter
+  - https://github.com/openebs/dynamic-nfs-provisioner
+  - https://github.com/openebs/openebs-k8s-provisioner (deprecated)
+  - https://github.com/openebs/openebs-docs (deprecated)
+  - https://github.com/openebs/maya (deprecated)
```
Comment (avishnu) [inline ROADMAP.md:100]: I guess, we can remove the deprecated ones from this list.

File: ROADMAP.md [hunk 9]
```diff
@@ -1,56 +1,109 @@
...
+- Backlogs
+  - Enhancements to OpenEBS CLI (openebsctl) for better troubleshooting OpenEBS components and fixing the errors
```
Comment (avishnu) [inline ROADMAP.md:102]: This can be removed IMO.

File: ROADMAP.md [hunk 10]
```diff
@@ -72,46 +125,17 @@ Note: OpenEBS follows a quarterly release cadence with a new minor release aroun
...
-  - Enhancements to OpenEBS CLI (openebsctl) for better troubleshooting OpenEBS components and fixing the errors
-
-## Near Term
-
-Typically the items under this category fall under next major release (after the current. e.g 4.0). At a high level, the focus is towards moving the beta engines towards stable by adding more automated e2e tests and updating the corresponding user and contributor documents. To name a few backlogs (not in any particular order) on the near-term radar, where we are looking for additional help: 
 
+## Old items (needs tidy-up)
 
-- Support for Mayastor Volume resize
+Typically the items under this category fall under next major releas.
```
Comment (avishnu) [inline ROADMAP.md:131]: release*

File: ROADMAP.md (hunk 1 above)
Comment (tiagolobocastro) [reply ROADMAP.md:3]: I see resolved but the typo is still here @orville-wright, did you perhaps forget to git push?

**Pull request closed without merging**